api/src/Entity/ActivityResponsible.php

```diff
@@ -33,7 +33,10 @@
     denormalizationContext: ['groups' => ['write']],
     normalizationContext: ['groups' => ['read']],
 )]
-#[UniqueEntity(fields: ['activity', 'campCollaboration'])]
+#[UniqueEntity(
```

Comment:
When we have multiple roles a person can have for one activity, maybe a camp member needs to have multiple roles for one activity.
But we can loosen the constraint when we need it.

Reply:
Agree. Should be changed when we implement multiple roles. For the moment it's not allowed.